sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -171,27 +171,19 @@ def set_all_properties(self, properties, value):
         properties.can_read = value
         properties.can_write = value
         properties.can_list = value
-        # try:
-        #     properties.teleport_enabled = value
-        # except AttributeError:
-        #     pass
         return properties
 
     def assert_all_properties(self, properties, value):
         assert properties.can_delete == value
         assert properties.can_read == value
         assert properties.can_write == value
         assert properties.can_list == value
-        # try:
-        #     assert properties.teleport_enabled == value
-        # except AttributeError:
-        #     pass
 
 
 # Moving this out of testcase so the fixture and individual tests can use it
 def import_image(repository, tags):
     mgmt_client = ContainerRegistryManagementClient(
-        DefaultAzureCredential(), os.environ["CONTAINERREGISTRY_SUBSCRIPTION_ID"]
+        DefaultAzureCredential(), os.environ["CONTAINERREGISTRY_SUBSCRIPTION_ID"], api_version="2019-05-01"
```

Comment:
just checking why you're specifying api version here. is this a test for an older api version?

Reply:
This is for the management plane client, I use mgmt-plane to load the registry for tests.